contrib_bots/lib/john.py

```diff
@@ -0,0 +1,118 @@
+# pip install chatterbot is necessary
+import json
+import os
+import time
+
+from random import choice
+
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+
+JOKES_PATH = os.path.expanduser('lib/John/jokes.json')
+DATABASE_PATH = os.path.expanduser('~/JohnBot/database.db')
+
+# Create a new instance of a ChatBot
+def CreateChatBot(nolearn):
```

Comment:
Have you considered creating the `DATABASE_PATH` here if it doesn't exists by this point?